x-pack/platform/plugins/shared/alerting/server/alerts_client/types.ts

```diff
@@ -130,6 +130,7 @@ export interface InitializeExecutionOpts {
   flappingSettings: RulesSettingsFlappingProperties;
   activeAlertsFromState: Record<string, RawAlertInstance>;
   recoveredAlertsFromState: Record<string, RawAlertInstance>;
+  executionUuid?: string;
```

Comment:
nit: maybe clearer to call this `previousExecutionUuid`?

Reply:
done

Reply:
I changed all this to `trackedExecutions`
We keep the last execution ids in that list. 
It is limited to `flapping.lookBackWindow`. The oldest execution uuid gets removed when the list reaches to the limit.